pkg/asset/manifests/nutanix/cloudproviderconfig.go

```diff
@@ -0,0 +1,104 @@
+package nutanix
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+)
+
+// Config of Nutanix cloud provider
+// ref: https://github.com/nutanix-cloud-native/cloud-provider-nutanix/blob/main/pkg/provider/config/config.go
+type NutanixCloudConfig struct {
+	PrismCentral         NutanixPrismEndpoint `json:"prismCentral"`
+	TopologyDiscovery    TopologyDiscovery    `json:"topologyDiscovery"`
+	EnableCustomLabeling bool                 `json:"enableCustomLabeling"`
+}
+
+type TopologyDiscovery struct {
+	// Default type will be set to Prism via the newConfig function
+	Type               TopologyDiscoveryType `json:"type"`
+	TopologyCategories *TopologyCategories   `json:"topologyCategories"`
+}
+
+type TopologyDiscoveryType string
+
+const (
+	PrismTopologyDiscoveryType      = TopologyDiscoveryType("Prism")
+	CategoriesTopologyDiscoveryType = TopologyDiscoveryType("Categories")
+)
+
+type TopologyInfo struct {
+	Zone   string `json:"zone"`
+	Region string `json:"region"`
+}
+
+type TopologyCategories struct {
+	ZoneCategory   string `json:"zoneCategory"`
+	RegionCategory string `json:"regionCategory"`
+}
+
+type NutanixPrismEndpoint struct {
```

Comment:
the linter doesn't like to see "Nutanix" at the beginning of exported values from this package because it considers the usage to be repetitive, eg a user would instantiate this struct by calling `nutanix.NutanixPrismEndpoint` (assuming a standard import of this package).

we should change this name to `PrismEndpoint` since we know it's for Nutanix by the fact that it resides in the `nutanix` pacakge.

Reply:
Fixed